slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -421,11 +472,16 @@ void SLAMServiceImpl::GetLatestSampledPointCloudMapString(
         float y_pos = -(pixel_y - painted_slices->origin.y()) * kPixelSize;
         // 2D SLAM so Z is set to 0
         float z_pos = 0;
+        Eigen::Vector3d map_point(x_pos, y_pos, z_pos);
+        auto rotated_map_point = pcdRotation * map_point;
 
         // Rotating coordinates to match slam service expectation (XZ plane)
-        viam::utils::writeFloatToBufferInBytes(data_buffer, x_pos);
-        viam::utils::writeFloatToBufferInBytes(data_buffer, z_pos);
-        viam::utils::writeFloatToBufferInBytes(data_buffer, y_pos);
+        viam::utils::writeFloatToBufferInBytes(data_buffer,
```

Comment:
Is this fixing a bug in the original endpoint? If so, what was the bug & what is the new intended behavior?

Reply:
new intended behavior is so if there are any other transformations needed for cartographer's endpoint that they get captured. Previous behavior had me manually rotating the cartographer map points(z_pos and y_pos being flipped). 

If we end up needing more changes to the map orientation I wanted to make it easier/clearer to capture them for both the Position and PointCloud endpoints. Can add a comment at the math in GetPointcloud so its clear that we are using the rotation here too